src/main/java/org/mybatis/spring/MyBatisSystemException.java

```diff
@@ -31,8 +31,8 @@ public class MyBatisSystemException extends UncategorizedDataAccessException {
 
   private static final long serialVersionUID = -5284728621670758939L;
 
-  public MyBatisSystemException(Throwable cause) {
-    super(null, cause);
+  public MyBatisSystemException(String msg, Throwable cause) {
```

Comment:
I suggest to add a new constructor with `msg` and `cause` arguments for backward-compatibility as API level. (And I prefer to deprecate the existing constructor).

Reply:
Hm..Okay.
In the one-arg constructor, what should we pass as the first argument of the two-arg constructor?

```java
public MyBatisSystemException(Throwable cause) {
    super(null, cause);
}
```
```java
public MyBatisSystemException(Throwable cause) {
    super(cause.getMessage(), cause);
}
```

Reply:
I think better `cause.getMessage()`.